Common/Data/Auxiliary/FactorFileRow.cs

```diff
@@ -296,7 +296,10 @@ public static FactorFileRow Parse(string line)
         public string ToCsv(string source = null)
         {
             source = source == null ? "" : $",{source}";
-            return $"{Date.ToString(DateFormat.EightCharacter)},{Math.Round(PriceFactor, 6).Normalize()},{Math.Round(SplitFactor, 7).Normalize()},{Math.Round(ReferencePrice, 2).Normalize()}{source}";
+            return $"{Date.ToString(DateFormat.EightCharacter)}," +
```

Comment:
has this branch been rebased on `master`? There are new functions in `master` for properly `ToString`-ing values using the invariant culture. This `Date.ToString` can be updated to `Date.ToStringInvariant`.

Reply:
@mchandschuh I will be fixing this very shortly. :+1: Thanks!